code/__DEFINES/mobs.dm

```diff
@@ -82,8 +82,6 @@
 #define UNCONSCIOUS	1
 #define DEAD		2
 
-#define check_tod(H) (!H.undefibbable && !(H.dead_ticks < TIME_BEFORE_DNR))
```

Comment:
Should have been `(!H.undefibbable && H.dead_ticks < TIME_BEFORE_DNR)`, that is defibabble and the time is still before DNR.

Reply:
Yeah, but check_tod was not a good solution IMO